lib/src/tag/tagger.dart

```diff
@@ -295,6 +299,14 @@ class Tagger {
     }
   }
 
+  /// Adds tags for Flutter plugins.
+  void flutterPluginTags(List<String> tags, List<Explanation> explanations) {
+    const _pluginTag = 'is:plugin';
+    if (_pubspec.hasFlutterPluginKey) {
```

Comment:
We have access to the pubspec already:
```suggestion
    final pubspec = _pubspecCache.pubspecOfPackage(packageName);
    if (pubspec.hasFlutterPluginKey) {
```